packages/client-sdk-nodejs/src/storage-client.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  AbstractStorageClient,
+  IStorageControlClient,
+  IStorageDataClient,
+} from '@gomomento/sdk-core/dist/src/internal/clients';
+import {IStorageClient} from '@gomomento/sdk-core/dist/src/clients/IStorageClient';
+import {StorageClientProps} from './storage-client-props';
+import {StorageClientPropsWithConfig} from './internal/storage-client-props-with-config';
+import {StorageControlClient} from './internal/storage-control-client';
+import {StorageDataClient} from './internal/storage-data-client';
+import {StorageConfiguration} from './config/storage-configuration';
+import {StorageConfigurations} from './index';
+
+export class StorageClient
```

Comment:
```suggestion
export class PreviewStorageClient
```

Reply:
👍 this is a very important change to make for now.

We also need to make sure we have docstrings explaining that this API is not fixed yet.